qiskit_finance/data_providers/random_data_provider.py

```diff
@@ -28,50 +26,62 @@ class RandomDataProvider(BaseDataProvider):
 
     def __init__(
         self,
-        tickers: Optional[Union[str, List[str]]] = None,
+        tickers: str | list[str] | None = None,
         start: datetime.datetime = datetime.datetime(2016, 1, 1),
         end: datetime.datetime = datetime.datetime(2016, 1, 30),
-        seed: Optional[int] = None,
+        seed: int | None = None,
     ) -> None:
         """
+        Initialize RandomDataProvider.
+
         Args:
-            tickers: tickers
-            start: first data point
-            end: last data point precedes this date
-            seed: optional random seed
+            tickers (str | list[str] | None): Tickers for the data provider.
+                Default is None, using ["TICKER1", "TICKER2"] if not provided.
+            start (datetime.datetime): Start date of the data.
+                Defaults to January 1st, 2016.
+            end (datetime.datetime): End date of the data.
+                Defaults to January 30th, 2016.
+            seed (int | None): Random seed for reproducibility.
         """
         super().__init__()
         tickers = tickers if tickers is not None else ["TICKER1", "TICKER2"]
+
         if isinstance(tickers, list):
             self._tickers = tickers
         else:
             self._tickers = tickers.replace("\n", ";").split(";")
-        self._n = len(self._tickers)
 
+        self._n = len(self._tickers)
         self._start = start
         self._end = end
         self._seed = seed
 
     def run(self) -> None:
         """
-        Generates data pseudo-randomly, thus enabling get_similarity_matrix
-        and get_covariance_matrix methods in the base class.
-        """
+        Generate pseudo-random stock market data.
 
+        Generates pseudo-random stock market data using normal distribution
+        and truncates values to zero after the first occurrence of zero.
+
+        Raises:
+            None
+        Returns:
+            None
```

Comment:
Unless it Raises something or Returns something other than None I would omit these to avoid cluttering the docs with info of little value. You can say returns None but we don't elsewhere in general afaik. Indeed https://google.github.io/styleguide/pyguide.html#doc-function-returns says `If the function only returns None, this section is not required. ` I just commented on this `run` but similar can be said of the others in regards the Return.

Reply:
Thanks! Just removed the instances where this statement appears.